### ISSUE TITLE
Unable to save lines or areas

### ISSUE THREAD

**Issue by noffle** [digidem/mapeo-desktop#54]
I noticed that points save fine, but areas and lines write this to the browser process log:

```
Failed to load resource: the server responded with a status of 404 (Not Found)
iD-patched.js:43220 PUT /api/0.6/changeset/create
iD-patched.js:43220 POST /api/0.6/changeset/11990380259834217655/upload
http://127.0.0.1:5000/api/0.6/changeset/11990380259834217655/upload   Failed to load resource: the server responded with a status of 400 (Bad Request)
iD-patched.js:43230 Uncaught TypeError: Cannot read property 'response' of undefined
```

**Comment by gmaclennan** [comment]
Yes hit that in the field and made a fix in osm-p2p-server v2.0.0-beta4 and just published, just dashing back to a meeting could you update Mapeo with this and kick off a new build?

Strange bug: When you create a changeset upload for OSM you create placeholder ids that are replaced on save, by convention they are negative. From what I read from the OSM docs I assumed that these placeholder ids must be unique within a changeset upload, but iD Editor can create duplicate ids within a changeset upload between types (nodes, ways). This may actually be a bug in iD editor, and not noticed before, since it only seems to come up on the very first save. For now I've loosened the checks in osm-p2p-server to only check uniqueness within types. https://github.com/digidem/osm-p2p-server/commit/3ad51b8d635ee402653c859f5093424da1342ccb

**Comment by noffle** [comment]
v1.2.11 in progress: https://github.com/digidem/mapeo-desktop/releases/tag/v1.2.11
